index.js

```diff
@@ -1,36 +1,58 @@
 import { ChatGPTAPI } from 'chatgpt'
-import  express from 'express'
-import * as dotenv from 'dotenv'
+import express from 'express'
+import dotenv from 'dotenv'
 dotenv.config()
 
-const app = express();
+const app = express()
 
 
+// Enable CORS middleware
+app.use(function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*')
```

Comment:
Later on this one we should change it / remove it, or at least only enable it in development mode. we can also add specific URLs to the allowed origins instead of using a wildcard like `*`